core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
When i add `lager:debug("DEBUG: ~p",[BindingProps]),` and restart whistle_apps it show me this:
```
Sep 24 15:39:19 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1070.0>) DEBUG: [[]]
Sep 24 15:39:19 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1093.0>) DEBUG: [[{type,<<"account">>}],[{db,<<"system_config">>}]]
Sep 24 15:39:19 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1110.0>) DEBUG: [[{type,<<"account">>}]]
Sep 24 15:39:19 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1140.0>) DEBUG: [[{type,<<"number">>}]]
Sep 24 15:39:20 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1194.0>) DEBUG: [[{type,<<"resource">>}],[{type,<<"number">>}],[{type,<<"dedicated_ip">>}]]
Sep 24 15:39:20 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1315.0>) DEBUG: [[{type,<<"account">>}],[{type,<<"user">>}],[{type,<<"device">>}],[{type,<<"parked_calls">>}],[{doc_id,<<"manual_presence">>}]]
Sep 24 15:39:21 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1497.0>) DEBUG: [[{type,<<"notification">>}]]
Sep 24 15:39:22 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1601.0>) DEBUG: [[{type,<<"account">>}],[{type,<<"connectivity">>}],[{type,<<"sys_info">>}],[{type,<<"number">>}]]
Sep 24 15:39:22 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.1701.0>) DEBUG: [[{db,<<"faxes">>},{type,<<"faxbox">>}]]
Sep 24 15:39:23 dev 2600hz[20989]: |undefined|wh_cache:135 (<0.2134.0>) DEBUG: [[{type,<<"limits">>}],[{type,<<"user">>}],[{type,<<"device">>}]]
```